src/Constraint/MultiConstraint.php

```diff
@@ -144,6 +144,57 @@ public function getUpperBound()
         return $this->upperBound;
     }
 
+    /**
+     * Tries to optimize the constraints as much as possible, meaning
+     * reducing/collapsing congruent constraints etc.
+     * Does not necessarily return a MultiConstraint instance if
+     * things can be reduced to a simple constraint
+     *
+     * @param ConstraintInterface[] $constraints A set of constraints
+     * @param bool                  $conjunctive Whether the constraints should be treated as conjunctive or disjunctive
+     *
+     * @return ConstraintInterface
+     */
+    public static function create(array $constraints, $conjunctive = true)
+    {
+        if (0 === count($constraints)) {
+            return new self($constraints, $conjunctive);
```

Comment:
Should that not be an EmptyConstraint?

Reply:
Imho, yes. However, `MultiConstraint` currently accepts an empty array as constructor argument and it's also part of the unit tests. Imho that's nonsense though so I think I'd add a check in the constructor for a non-empty array and return an `EmptyConstraint` here. That would be the proper way to go imho. Agree?

Reply:
Agreed yes 👍

Reply:
Done :)